### PR TITLE
fix-3608: there is no log with two parameters 

### PR DESCRIPTION
**Pull request by YANGGMM**
## What type of PR is this?

- [ ] API-change
- [x] BUG
- [ ] Improvement
- [ ] Documentation
- [ ] Feature
- [ ] Test and CI
- [ ] Code Refactoring

## Which issue(s) this PR fixes:

issue # 3608

## What this PR does / why we need it:
there is no log with two parameters

### CODE REVIEW

File: pkg/sql/plan/function/builtin/unary/math.go [hunk 1]
```diff
@@ -91,16 +91,16 @@ func Log(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
 	vals := vs[0].Col.([]float64)
 	for i := range vals {
 		if vals[i] == float64(1) {
-			return nil, errors.New("", "Logarithm function base cannot be 1")
+			return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function base cannot be 1")
```
Comment (fengttt) [inline pkg/sql/plan/function/builtin/unary/math.go:94]: This won't work, you need to merge.

File: pkg/sql/plan/function/builtin/unary/math.go [hunk 2]
```diff
@@ -91,16 +91,16 @@ func Log(vs []*vector.Vector, proc *process.Process) (*vector.Vector, error) {
...
 	}
 	v1, err := math1([]*vector.Vector{vs[0]}, proc, momath.Ln)
 	if err != nil {
-		return nil, err
+		return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function input cannot be non positive")
 	}
 	v2, err := math1([]*vector.Vector{vs[1]}, proc, momath.Ln)
 	if err != nil {
-		return nil, err
+		return nil, moerr.New(moerr.INVALID_ARGUMENT, "Logarithm function input cannot be non positive")
```
Comment (fengttt) [inline pkg/sql/plan/function/builtin/unary/math.go:103]: seesm return nil, err is the right thing.  Why moerr.New again?

File: pkg/sql/plan/function/builtin/unary/math.go (hunk 1 above)
Comment (YANGGMM) [reply pkg/sql/plan/function/builtin/unary/math.go:94]: ok, thanks

File: pkg/sql/plan/function/builtin/unary/math.go (hunk 2 above)
Comment (YANGGMM) [reply pkg/sql/plan/function/builtin/unary/math.go:103]: i did't get the newest codes when i fixed the problem.

**Pull request closed without merging**